### ISSUE TITLE
java closes the connection and closes the port

### ISSUE THREAD

**Issue by hansw-nlo** [languagetool-org/languagetool#9090]
Hi all, 

Love this tool. I downloaded the tool for at home and configured it to run on my own desktop which is running FreeBSD.

The only problem I see is it closes the connection when the pc is asleep and or after some given time.

**Comment by danielnaber** [comment]
You mean you start LT with a command like `java -cp languagetool-server.jar org.languagetool.server.HTTPServer` and it stops after some time? I'm not aware of such a behavior - are you sure it's not the OS or some other process that kills the LT process?

**Comment by hansw-nlo** [comment]
I did start it that way yes, from a script. 

This is the only software that somehow vanishes. Are there specific log files to watch?

**Comment by danielnaber** [comment]
> Are there specific log files to watch?

No, LT prints to STDOUT.

**Comment by hansw-nlo** [comment]
Ok, in that case I will forward that to a log file. Thanks so far.

**Comment by hansw-nlo** [comment]
I did notice some other things that might be wanted. Would love to see it forking the instance, so it does not run as root or as an existing user. That way it might be able to start as root, fork the instance to some user and then destroy the root process.

**Comment by hansw-nlo** [comment]
Just wanted to let you all know that the software runs fine now. Have tested it all day long where the pc did not hibernate but did disconnect the account for power saving. Once I enabled it again I did not have to restart the service. 

Logging to a file might indeed be a good idea. I can start it from the command line to see the logs, but it might be good to have logging enabled. 

Forking the software to a user without any rights would be high on my priority list. Hope you people will enable that.
